docs/modules/ROOT/pages/list-of-metrics.adoc

```diff
@@ -2512,3 +2512,30 @@ Based on your latency tolerance in your business use case, you can define a thre
 |Number of entry update events
 |===
 ====
+
+.User Code Namespaces
+[%collapsible]
+====
+[cols="4,1,6a"]
+|===
+| Name
+| Unit
+| Description
+
+|`user.code.namespace.updateTime`
+|ms
+|Update time of a namespace configuration
```

Comment:
```suggestion
|Update time of a user code namespace configuration
```

Reply:
Not pushing this - it is in a UCN section, so it should be clear enough, but there are other namespaces.....

Leave as is, or change all occurrences to user code namespaces just to ensure it is clear to folk scanning. Whichever you feel best.

Reply:
Good spot, thanks!